src/commands/dumpDb.ts

```diff
@@ -1,17 +1,18 @@
 import { DumpDbOptions, Registry } from "../types";
-import { DBService } from "../utils";
+import { logger, DBService } from "../utils";
 
 /**
  * Dump the database as JSON to STDOUT for a given chain ID
  * @param options A dict, but essentially just the chainId
  */
 export async function dumpDb(options: DumpDbOptions) {
+  const log = logger.getLogger("commands:dumpDb");
```

Comment:
Would it make sense to create the log instance in the module scope instead or getting it every time the fn runs?

Might not matter than much, actually.